src/Datadog.Trace.Core/Datadog.Trace.Core.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+  
+    <!-- NuGet -->
+    <Version>1.1.0</Version>
```

Comment:
Leaving a comment here as a reminder to change this back to 1.0.0 before merging.